oeps/oep-0011.rst

```diff
@@ -0,0 +1,240 @@
+   implemented to handle as much of the "asset pipeline" as possible,
+   rather than passing this responsibility on to Django.
+
+7. **JavaScript dependencies should be managed with ES2015 Modules**
+
+   **Rationale**: JavaScript module systems allow front end code to specify
+   its dependencies and be grouped into bundles that minimize the assets
+   needed to provide page functionality. The most prevalent module syntax
+   is currently `ES2015 Modules`_, which should be adopted everywhere
+   edX code is written to the ES2015 spec or later.
+
+   **Exception**: Much of edX's existing (ES5) JavaScript uses the older
+   `AMD Modules`_ syntax for modules. AMD Modules are interoperable
+   with ES2015 Modules if Webpack is used for bundling, so AMD is an
+   acceptable module definition if the code must remain ES5.
+
+8. **CSS should be generated using Sass**
+
+   **Rationale**: Sass's SCSS syntax is an extension of CSS that adds power
+   and elegance to the basic language. It makes the maintenance of large
+   CSS files more manageable though the ability to use variables, mixins,
+   imports and more. In particular, it makes theming possible with the
+   ability to override variables that define colors, fonts etc.
+
+   You can find out more about Sass in the official `Sass documentation`_.
+
+Rejected Alternatives
+=====================
+
+.. _Reasons for rejecting Angular:
+1. **Angular**
+
+   `Angular`_ was a popular web framework in its 1.x iteration, and its
+   maintainer (Google) recently released Angular 2. Angular 1 was not considered
+   due to its support status. Angular 2 was rejected due to its opinionated
+   characteristics including its use of TypeScript (a compile-to-JavaScript
+   language) and the difficulty of incrementally integrating Angular into an
+   existing application (e.g., alongside legacy Backbone). Consensus in the
+   front end community is that Angular has a higher learning curve than
+   alternatives like React, which matches poorly with edX's limited front end
+   developers and larger community of infrequent JavaScript contributors.
+
+.. _Reasons for rejecting Polymer:
+2. **Polymer**
+
+   `Polymer`_ is a framework built around `Web Components`_, a set of draft
+   extensions to the HTML and DOM specs for UI encapsulation. Web Components
+   are a very interesting area of emergent Web technology, but are not yet
+   finalized as a spec or supported in many browsers. Because of this, the
+   Polymer community is small and performance issues exist. As Web Components
+   become more mature, edX may reevaluate their use, either with Polymer or
+   without.
+
+.. _Reasons for rejecting CoffeeScript:
+3. **CoffeeScript**
+
+   The Open edX codebase made heavy use of CoffeeScript for several years,
+   but its use at edX has now been officially deprecated. Most of the language
+   benefits of CoffeeScript were made available in Javascript as of the ES2015 spec,
```

Comment:
Nit: ``Javascript`` > ``JavaScript`` as it seem to be the format of the word that you chose.